data/tld-!cn

```diff
@@ -468,6 +470,15 @@ teva # Teva Pharmaceutical Industries Limited
 toray # Toray Industries, Inc.
 weather # The Weather Channel LLC
 
+# Sex & Porn
+adult # Adult content
+gay # Homosexuality
+lgbt # Lesbian, Gay, Bisexual, Transgender
```

Comment:
might not all the sites under `.lgbt` are about porn.
I would suggest we put that TLD under no categories, until we have built up good knowledge about it.

Reply:
> might not all the sites under `.lgbt` are about porn. I would suggest we put that TLD under no categories, until we have built up good knowledge about it.

Ok, I think it can be move to `Miscellaneous Domains`.

Reply:
> > might not all the sites under `.lgbt` are about porn. I would suggest we put that TLD under no categories, until we have built up good knowledge about it.
> 
> Ok, I think it can be moved to `Miscellaneous Domains`.

That would be a good start. Mark this conversation as resolved when you finished fixing that.